woosmapgeofencingcore/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.webgeoservices.woosmapgeofencingcore">
+<manifest xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android">
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.location.GPS_ENABLED_CHANGE" />
     <uses-permission android:name="android.permission.VIBRATE" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE_LOCATION" />
```

Comment:
Introduced new permission. What was its expected purpose in SDK?

Reply:
https://stackoverflow.com/questions/76943771/foreground-service-crashing-on-android-14

Reply:
We missed adding this line to support Android 14.
```
Service.startForeground(0, notification, FOREGROUND_SERVICE_TYPE_LOCATION)
```

Reply:
Not needed since we are starting the service by invoking

```
startForegroundService( new Intent( getApplicationContext(), LocationUpdatesServiceCore.class ) );
```